xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/junit5/servletEngine/ServletContainerExecutor.java

```diff
@@ -168,6 +170,71 @@ private void startContainer(GenericContainer servletContainer)
         start(servletContainer, this.testConfiguration);
     }
 
+    private void manageDefaultServletEngine(TestConfiguration testConfiguration)
```

Comment:
I don't understand the name of the method when all it does is throw an exception.